docs/config.md

```diff
@@ -1554,7 +1560,33 @@ paginate = "never"
 pager = "delta"
 ```
 
-Condition keys:
+#### Using multiple files
+`--when` can also be used on the top level of a TOML file, which is convenient
+when splitting your config across multiple files using `JJ_CONFIG`.
+The behavior of conditions are the same as when using `[[--scope]]` tables.
+
```

Comment:
nit: maybe better to mention the load order again? If `work.toml` were `a.toml`, it would be overridden by `config.toml`.

Reply:
Added a sentence to point this out.